src/main/java/seedu/address/model/person/Contact.java

```diff
@@ -16,6 +17,8 @@
  */
 public class Contact {
 
+    private static String illegalOperationMessage = "Contact cannot have child contacts";
```

Comment:
Since this is a constant, it should be formatted in all caps and set to final.

Reply:
Done, but the field is still private.